### ISSUE TITLE
stack.pop returning garbage (UnicodeEncodeError: 'ascii' codec can't encode ..)

### ISSUE THREAD

**Issue by rhoerbe** [kivy/pyjnius#309]
A simple test program with py3.4 and Cython 0.26 stoppend working with Cython 0.27.3

    Stack = autoclass('java.util.Stack')
    stack = Stack()
    stack.push('OK')
    print(stack.pop())

   *** UnicodeEncodeError: 'ascii' codec can't encode character '\xe5' in position 1: ordinal not in range(128)

I will post a repo to reproduce the issue

**Comment by rhoerbe** [comment]
could not reproduce the issue in an stripped down environment - need to do more research

**Comment by Ariel-Lieberman** [comment]
Maybe a work around ...

```
Stack = autoclass('java.util.Stack')
jString = autoclass('java.lang.String')
stack = Stack()
stack.push(jString('OK'))
print(stack.pop())
```

**Comment by Arnie97** [comment]
Seems to be a duplicate of #300.

**Comment by KeyWeeUsr** [comment]
Not sure about whether this is 100% duplicate or not. Anyway, if the issue still persists, open a new issue. Probably resolved by #285.
